app/models/user.rb

```diff
@@ -77,6 +78,10 @@ def print_id
     %(#{print} - #{id})
   end
 
+  def print_email
+    %(#{print} <#{email}>)
+  end
+
```

Comment:
Here you have five functions (to_s, print, full_name, print_id, print_email) which are doing essentially the same thing. Could some of them be removed or perhaps moved to a view helper somewhere?